### PR TITLE
Add plugin to generate RSS Feeds

### LINKED ISSUE #58: Generate RSS Feed data for RSS Reader
# Description
* 增加订阅 feed 信息

# Solution
使用插件

https://github.com/hexojs/hexo-generator-sitemap

自动构建后，从 https://flossway.github.io/flossway/sitemap.xml 获得 Feeds

### PR DESCRIPTION
**Pull request by hailiang-wang**
* Add plugin to generate RSS Feeds

This closes #58.

### CODE REVIEW

**Comment by hailiang-wang** [comment]
By default, in the preview mode, the URLs in the sitemap.xml is invalid due to the path. It should be fine in the production build.


image

**Comment by hailiang-wang** [comment]
用 Rss Reader，方便订阅多个渠道的消息，可以更高效的做信息的处理，比在微信上更适合工作。所以增加了这个功能。

File: _config.next.yml [hunk 1]
```diff
@@ -886,4 +886,16 @@ vendors:
...
+## https://github.com/hexojs/hexo-generator-sitemap
+sitemap:
+  path: 
+    - sitemap.xml
+    - sitemap.txt
+  template: ./assets/tpls/sitemap_template.xml
+  template_txt: ./assets/tpls/sitemap_template.txt
+  rel: false
+  tags: true
+  categories: true
```
Comment (tisonkun) [inline _config.next.yml:901]: You don't have to make change on the theme config file. Please revert these.

File: _config.yml [hunk 2]
```diff
@@ -126,4 +126,16 @@ giscus:
...
+## https://github.com/hexojs/hexo-generator-sitemap
+sitemap:
+  path: 
+    - sitemap.xml
+    - sitemap.txt
+  template: ./assets/tpls/sitemap_template.xml
+  template_txt: ./assets/tpls/sitemap_template.txt
+  rel: false
+  tags: true
+  categories: true
```
Comment (tisonkun) [inline _config.yml:141]:
```suggestion
  categories: true

```

File: _config.yml [hunk 3]
```diff
@@ -126,4 +126,16 @@ giscus:
...
+  theme: light
+
+# Sitemap Generator for RSS Reader
+## https://github.com/hexojs/hexo-generator-sitemap
+sitemap:
+  path: 
+    - sitemap.xml
+    - sitemap.txt
+  template: ./assets/tpls/sitemap_template.xml
+  template_txt: ./assets/tpls/sitemap_template.txt
```
Comment (tisonkun) [inline _config.yml:138]: I'd prefer to place these files just under:

```yml
  template: ./sitemap_template.xml
  template_txt: ./sitemap_template.txt
```

File: assets/tpls/sitemap_template.xml [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+  {% for cat in categories %}
+  <url>
+    <loc>{{ cat.permalink | uriencode }}</loc>
+    <lastmod>{{ sNow | formatDate }}</lastmod>
+    <changefreq>weekly</changefreq>
+    <priority>0.2</priority>
+  </url>
+  {% endfor %}
+</urlset>
```
Comment (tisonkun) [inline assets/tpls/sitemap_template.xml:40]:
```suggestion
</urlset>

```

**Review by tisonkun — COMMENTED** [review]
Generally looks good. Thank you!

Comments inline.

**Review by tisonkun — APPROVED** [review]
LGTM. Thanks for your contribution! Merging...

**Pull request merged**